docs/manual/external-tools.tex

```diff
@@ -537,6 +538,34 @@ \section{Javac compiler\label{javac-installation}}
 % \end{itemize}
 
 
+\section{Lombok\label{lombok}}
+
+Project Lombok (\url{https://projectlombok.org/}) is ``a java library that
+automatically plugs into your editor and build tools, spicing up your java.''
+Starting with version 1.18.3 it is generally easy to combine Lombok and the
+Checker Framework, because type annotations are e.g. copied from fields to
+generated setter/getter/builder methods.
+
+Many standard Nullable/NonNull annotations are automatically copied, however the
+Checker Framework annotations in \<org.checkerframework> are currently not copied.
```

Comment:
(I presume you have a pull request open to add all the Checker Framework annotations?)

Reply:
`BASE_COPYABLE_ANNOTATIONS` requires fully-qualified class names. I do not expect that Lombok would accept a PR that adds all the 20+ Checker Framework type annotations.

Reply:
Why don't you think they would accept that?  It would help users and wouldn't harm maintainability of Lombok beyond the current list.